docs/sources/datasources/grafana-pyroscope.md

```diff
@@ -61,26 +63,27 @@ The query editor gives you access to a profile type selector, a label selector,
 
 To access the query editor:
 
-1. Sign into Grafana or Grafana Cloud.
-1. Select your Pyroscope data source.
-1. From the menu, choose **Explore**.
+1.  Sign into Grafana or Grafana Cloud.
+1.  Select your Pyroscope data source.
+1.  From the menu, choose **Explore**.
 
-1. Select a profile type from the drop-down menu.
+1.  Select a profile type from the drop-down menu.
 
     {{< figure src="/media/docs/pyroscope/query-editor/select-profile.png" class="docs-image--no-shadow" max-width="450px" caption="Profile selector" >}}
 
-1. Use the labels selector input to filter by labels. Pyroscope uses similar syntax to Prometheus to filter labels.
-Refer to [Pyroscope documentation](https://grafana.com/docs/pyroscope/latest/) for available operators and syntax.
+1.  Use the labels selector input to filter by labels. Pyroscope uses similar syntax to Prometheus to filter labels.
+    Refer to [Pyroscope documentation](https://grafana.com/docs/pyroscope/latest/) for available operators and syntax.
+
+        While the label selector can be left empty to query all profiles without filtering by labels, the profile type or app must be selected for the query to be valid.
 
-    While the label selector can be left empty to query all profiles without filtering by labels, the profile type or app must be selected for the query to be valid.
+        Grafana doesn't show any data if the profile type or app isn’t selected when a query runs.
 
-    Grafana doesn't show any data if the profile type or app isn’t selected when a query runs.
-  ![Labels selector](/media/docs/pyroscope/query-editor/labels-selector.png 'Labels selector')
+    ![Labels selector](/media/docs/pyroscope/query-editor/labels-selector.png 'Labels selector')
 
-1. Expand the **Options** section to view **Query Type** and **Group by**.
-  ![Options section](/media/docs/pyroscope/query-editor/options-section.png 'Options section')
+1.  Expand the **Options** section to view **Query Type** and **Group by**.
+    ![Options section](/media/docs/pyroscope/query-editor/options-section.png 'Options section')
 
-1. Select a query type to return the profile data which can be shown in the [Flame Graph][flame-graph], metric data visualized in a graph, or both. You can only select both options in a dashboard, because panels allow only one visualization.
+1.  Select a query type to return the profile data which can be shown in the [Flame Graph][flame-graph], metric data visualized in a graph, or both. You can only select both options in a dashboard, because panels allow only one visualization.
```

Comment:
The spacing in this section looks off.

There are two spaces after each list item (which seems to render fine but is unusual).

Reply:
I'm not sure why. I'll check the formatting and fix it. Thank you.

Reply:
It should be good now.